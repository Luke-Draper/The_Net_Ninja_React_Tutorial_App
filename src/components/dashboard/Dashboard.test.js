import React from "react";
import Dashboard from "./Dashboard";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("Dashboard renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<Dashboard />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
