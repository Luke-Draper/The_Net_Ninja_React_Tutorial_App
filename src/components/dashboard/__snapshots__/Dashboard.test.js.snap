// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders without crashing 1`] = `
<div
  className="dashboard container"
>
  <div
    className="row"
  >
    <div
      className="col s12 m6"
    >
      <div
        className="project-list section"
      >
        <div
          className="card z-depth-0 project-summary"
        >
          <div
            className="card-content grey-text text-darken-3"
          >
            <span
              className="card-title"
            >
              Project Title
            </span>
            <p>
              Posted by Luke Draper
            </p>
            <p
              className="grey-text"
            >
              3rd September, 2018
            </p>
          </div>
        </div>
        <div
          className="card z-depth-0 project-summary"
        >
          <div
            className="card-content grey-text text-darken-3"
          >
            <span
              className="card-title"
            >
              Project Title
            </span>
            <p>
              Posted by Luke Draper
            </p>
            <p
              className="grey-text"
            >
              3rd September, 2018
            </p>
          </div>
        </div>
        <div
          className="card z-depth-0 project-summary"
        >
          <div
            className="card-content grey-text text-darken-3"
          >
            <span
              className="card-title"
            >
              Project Title
            </span>
            <p>
              Posted by Luke Draper
            </p>
            <p
              className="grey-text"
            >
              3rd September, 2018
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col s12 m5 offset-m1"
    >
      <div>
        <p>
          Notifications
        </p>
      </div>
    </div>
  </div>
</div>
`;
